src/binding.c

```diff
@@ -56,10 +56,20 @@ int bindInput(char* eventPath)
     return EXIT_SUCCESS;
 }
 
+/**
+ * Releases the input device.
+ * */
+int release_input(char* eventPath)

```

Comment:
Should this write a message to stdout with the unused eventPath parameter like is done in bind_input()?

Reply:
Added